tests/commands/init_test.py

```diff
@@ -10,18 +11,18 @@
 
 class TestLookupCommand(unittest.TestCase):
 
-    def test_1(self):
+    def test_look_up_save_attachment_command_in_thread_mode(self):
         cmd, parser, kwargs = lookup_command('save', 'thread')
         # TODO do some more tests with these return values
         self.assertEqual(cmd, SaveAttachmentCommand)
+        self.assertIsInstance(parser, argparse.ArgumentParser)
+        self.assertDictEqual(kwargs, {})
 
 
 class TestCommandFactory(unittest.TestCase):
 
-    def test_2(self):
+    def test_create_save_attachment_command_with_arguments(self):
         cmd = commandfactory('save --all /foo', mode='thread')
-        self.assertEqual(
-                cmd.__repr__()[:56],
-            "<alot.commands.thread.SaveAttachmentCommand object at 0x")
+        self.assertIsInstance(cmd, SaveAttachmentCommand)
```

Comment:
I like all of the changes in this patch, but could you squash them backinto the original commits?